docs/testing.md

```diff
@@ -164,8 +171,54 @@ the `deploytool` flag to the standard `make` commands.
 A large set of verifications for the Operator and the resulting Submariner
 deployment will automatically run during and after the deployment.
 
+
+#### Reloading your code changes
+During the development of new features you may want to compile submariner and push the images
+into the local registry used by the kind clusters. You can use the reload-images make target
+for that.
+
+This target depends on the build and packaging of the container images. It will push
+the new images to the local registry and restart all the services (gateway, routeagent, globalnet).
+
+```bash
+make reload-images
+```
+
+If you are working on a specific service, you can specify to only restart that service, for example:
+```bash
+make reload-images restart=gateway
+```
+
+If you don't want to restart any service because you plan to restart specific pods in specific clusters
+manually, then run:
+```bash
+make reload-images restart=none
+```
+
+#### Re-running e2e tests after your code changes
+Once your kind virtual clusters and submariner are deployed, you can re-run e2e just by repeating the `make e2e` call.
+Deployment and install will be avoided, hence the existing environment will remain as is.
+If you need to load new container images, please check `make reload-images` from the previous section.
+
+In case you want to re-run just the tests:
+```bash
+make e2e
+```
+
+In case you want to use updated images and re-run the tests:
+```bash
+make reload-images e2e
+```
+
+You can focus the e2e tests on specific tags by using the focus makefile env. The following
+example runs only the tests tagged as [redundancy].
+
+```bash
+make e2e focus=redundancy
+```
+
```

Comment:
Probably we want to remove this and link to the website.

Reply:
Yeah I wrote regarding that in the original commit message "which needs to be
ported to the website, but thats out of scope for this change"